requirements.txt

```diff
@@ -1,24 +1,6 @@
-## The following combination still work, but later versions of 1.x were broken
-# due to their upstream package brought in a breaking change
-# https://github.com/pallets/markupsafe/issues/286
-# Flask==1.1.4
-# werkzeug==1.0.1
-# Jinja2==2.11.3
-# MarkupSafe==2.0.1
```

Comment:
This specific combo seems fairly odd so I vote for removing the comment, but let me know if you want to keep in.

Reply:
OK, let's remove them.